devices/baseEstimatorV1/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+set(iDynTree_REQUIRED_VERSION 0.11.0)
+
+set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
+
+option(ENABLE_RPATH "Enable RPATH for this library" ON)
+mark_as_advanced(ENABLE_RPATH)
+include(AddInstallRPATHSupport)
+add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}"
+                          LIB_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+                          INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}"
+                          DEPENDS ENABLE_RPATH
+                          USE_LINK_PATH)
+
+find_package(Eigen3 REQUIRED)
+find_package(iDynTree REQUIRED)
+if(${iDynTree_VERSION} VERSION_LESS ${iDynTree_REQUIRED_VERSION})
+    message(FATAL_ERROR "iDyntree version ${iDynTree_VERSION} not sufficient, at least version ${iDynTree_REQUIRED_VERSION} is required.")
+endif()
```

Comment:
As before, why not simply `find_package(iDynTree 0.11 REQUIRED)`?

Reply:
Addressed in 82d978f1892d49beb5bdc29df0f343a153b2678a